python/kserve/requirements.txt

```diff
@@ -9,14 +9,14 @@ minio>=4.0.9,<7.0.0
 google-cloud-storage>=1.41.1
 adal>=1.2.2
 table_logger>=0.3.5
-numpy~=1.19.2
+numpy~=1.21.5
 azure-storage-blob==12.8.1
 azure-identity>=1.6.0
 cloudevents>=1.2.0
 avro>=1.10.1
 boto3==1.20.24
 psutil>=5.0
-ray[serve]==1.9.0
+ray[serve]==1.9.2
```

Comment:
this is the value in the latest version of this file in kserve ([link](https://github.com/kserve/kserve/blob/7e74dca4421af26cfbe96dccdf3806e77a63c1b3/python/kserve/requirements.txt#L20))